stylesheets/utils/utility.css

```diff
@@ -68,4 +68,7 @@ img {
   max-width: 100%;
   height: auto;
 }
+.shadow-lg{
+  box-shadow: #00000059 0px 5px 15px;
+}
 
```

Comment:
You can make variable for box -shadow as well. Rest your code is so clean and tidy.. Good to go!!

Reply:
Sure.